plugins/declarative/src/voicecallaudiorecorder.cpp

```diff
@@ -352,8 +360,9 @@ void VoiceCallAudioRecorder::terminateRecording()
             }
         }
 
-        const QString fileName(output->fileName());
-        output->close();
+        QString fileName(output->fileName());
+        fileName.chop(4); // remove .tmp
```

Comment:
nit: afaict it is not possible to end here without .tmp having been added, but I'd still be tempted to check that it actually has .tmp at end...

Reply:
I can add a sanity check.